extension/scripts/features/faction-member-info/ttFactionMemberInfo.js

```diff
@@ -0,0 +1,104 @@
+"use strict";
+
+(async () => {
+	if (!isOwnFaction()) return;
+
+	const feature = featureManager.registerFeature(
+		"Member Info",
+		"faction",
+		() => settings.pages.faction.memberInfo,
+		addListener,
+		addInfo,
+		removeInfo,
+		{
+			storage: ["settings.pages.faction.memberInfo"],
+		},
+		() => {
+			if (!hasAPIData() || !hasFactionAPIAccess()) return "No API access!";
+		},
+		{ liveReload: true }
+	);
+
+	function addListener() {
+		CUSTOM_LISTENERS[EVENT_CHANNELS.FACTION_INFO].push(async () => {
+			if (!feature.enabled()) return;
+
+			await addInfo(true);
+		});
+	}
+
+	async function addInfo(force) {
+		if (!force) return;
+		if (document.find(".tt-money-balance, .tt-points-balance")) return;
+
+		await requireElement(".members-list .table-body > li");
+
+		let donations;
+		if (ttCache.hasValue("faction-members-donations", userdata.faction.faction_id)) {
+			donations = ttCache.get("faction-members-donations", userdata.faction.faction_id);
+		} else {
+			donations = (
+				await fetchData("torn", {
+					section: "faction",
+					selections: ["donations"],
+					silent: true,
+					succeedOnError: true,
+				})
+			).donations;
+
+			ttCache.set({ [userdata.faction.faction_id]: donations }, TO_MILLIS.SECONDS * 30, "faction-members-donations").then(() => {});
+		}
+
+		document.findAll(".members-list .table-body > li").forEach((li) => {
+			const userID = li.find(".user.name").dataset.placeholder.match(/(?<=\[)\d+(?=]$)/g)[0];
+			if (donations[userID]) {
+				if (donations[userID].money_balance) {
+					if (li.nextSibling?.className?.includes("tt-last-action")) {
+						li.nextSibling.insertAdjacentElement(
```

Comment:
You are adding the same element in both cases. Either make the element beforehand, or only use it once and make a variable to find the parent (`li` or `li.nextSibling`)